vendor/bat-native-ledger/src/bat_helper.cc

```diff
@@ -2618,6 +2619,25 @@ uint64_t getMediaDuration(const std::map<std::string, std::string>& data,
   return duration;
 }
 
+// static
+bool isValidLeadingUrlSubString(
+    const std::string& url_to_validate,
+    const std::string& to_match) {
+  GURL gurl(url_to_validate);
```

Comment:
you should always check `is_valid` or you can get a DCHECK error in debug. I would return `false` if the `url_to_validate` isn't a valid url

Reply:
The first operation after initializing is checking `is_valid` here: https://github.com/brave/brave-core/pull/1792/files/cc8f8ff462f99b0491d65668c5cb16a546a7c597#diff-22bdd7998eb0956f61ab4a5ac07024e3R2631
Should I be doing this a different way?

Reply:
no, I didn't notice that. Thanks.